app/javascript/components/admin/instances/actions/delete.js

```diff
@@ -0,0 +1,65 @@
+import axios from 'axios'
+
+export default {
+  name: 'Delete',
+
+  props: {
+    instance: Object
+  },
+
+  methods: {
+    deleteInstance () {
+      this.processing = true
+
+      this.status = {
+        message: 'deleting',
+        level: 'warning',
+        icon: 'fa fa-spin fa-spinner'
+      }
+
+      this.button = {
+        text: 'Deleting...',
+        disabled: true,
+        icon: 'fa fa-spin fa-spinner'
+      }
+
+      this.$emit('updateStatus', { status: this.status, processing: this.processing })
+
+      axios.post(`/admin/instances/${this.instance.id}/delete.json`)
+        .then(response => {
+          this.status = response.data.status
+          this.processing = false
+
+          this.button = {
+            text: 'Delete',
+            disabled: false
+          }
+
+          this.$emit('updateStatus', { status: this.status, processing: this.processing })
+        })
+        .catch(err => {
+          alert(err)
```

Comment:
avoid alerts

Reply:
Just did this for debugging, forgot to remove.

Reply:
there is one in app/javascript/components/admin/instances/actions/restart.js